src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: AvenirB;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  margin: 20px 8px 8px;
+}
+
+.searchBar {
+  box-sizing: border-box;
+  width: 100%;
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  height: 70px;
+  font-size: 16px;
+  font-family: AvenirB, sans-serif, html;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position-x: 26px;
+  background-position-y: 26px;
+  padding-left: 62px;
+  font-weight: 300;
+  margin-bottom: 20px;
+  color: rgba(61, 78, 97, 1);
+}
+
+.searchBar:hover {
+  left: 0%;
+  right: 0%;
+  top: 0%;
+  bottom: 0%;
+  border: 1px solid #e1e7ed;
```

Comment:
looks like you already have those rules inside `.searchBar`, do you really need them here too?